content/source/docs/enterprise/getting-started/workspaces.html.md

```diff
@@ -128,7 +130,7 @@ Most of your time in TFE is spent in two views:
 
 * The workspace list. Use this to get an overview of the workspaces you're responsible for, and to navigate between workspaces.
 
-    ![main page](./images/work-mainpage.png)
+    ![main page](../workspaces/images/index-list.png)
```

Comment:
This screenshot actually needs to be updated again (sorry) - the status names have changed, and the "needs attention/no changes" issue that this shot shows is now resolved.